webpack.browser.config.js

```diff
@@ -19,18 +22,34 @@ module.exports = {
 		],
 	},
 	plugins: [
-		new LodashModuleReplacementPlugin( {
-			paths: true,
-			shorthands: true,
-		} ),
 		new webpack.DefinePlugin( {
-			'process.env.NODE_ENV': JSON.stringify( 'production' ),
+			'process.env.NODE_ENV': JSON.stringify( env ),
 		} ),
-		new webpack.optimize.ModuleConcatenationPlugin(),
-		new UglifyJsPlugin(),
 	],
 	resolve: {
 		extensions: [ '.js', '.jsx' ],
 		modules: [ path.resolve( __dirname ), path.resolve( __dirname, 'node_modules' ) ],
 	},
 };
+
+switch ( env ) {
+	case 'development':
+		config.devtool = 'source-map';
+		config.devServer = {
+			contentBase: path.resolve( __dirname, 'targets/browser' ),
+			publicPath: '/',
+			port: 9000,
+		};
+		break;
+
+	case 'production':
+		config.plugins.push( new webpack.optimize.ModuleConcatenationPlugin() );
+		config.plugins.push( new UglifyJsPlugin() );
+		config.plugins.push( new LodashModuleReplacementPlugin( {
```

Comment:
Same here: no longer use `lodash-webpack-plugin`.

Reply:
Will remove it.